jekyll/_cci2/gh-bb-integration.md

```diff
@@ -86,24 +86,51 @@ ERROR: Repository not found.
 Permission denied (publickey).
 ```
 
+## Machine User Keys
+
+For security reasons,
+consider creating a machine user
+for your CircleCI projects.
+A [machine user](https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users) is a GitHub user
+that you create for running automated tasks.
+By using the SSH key of a machine user,
+you allow anyone with repository access
+to build, test, and deploy the project
+without sacrificing security.
```

Comment:
'without sacrificing security' seems vague, I'm not sure what it means.

Reply:
Cool, I can flesh this one out.